app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -139,6 +147,12 @@ private void setUpDb(){
             Rooms rooms = ApplicationState.getInstance(this).getRooms();
             appDb.roomDao().insertAll(rooms.getRooms());
 
+            //Get shuttle schedule
+            Shuttles shuttles = ApplicationState.getInstance(this).getShuttles();
+            appDb.shuttleDao().insertAll(shuttles.getShuttles());
+
+            List<Shuttle> sh = appDb.shuttleDao().getScheduleByCampus("SGW");
```

Comment:
Why are you retrieving the data for the shuttles in the SetUpDb() method? Is there a particular reason ? Because the  purpose of the setUpDb() method is just for inserting data inside the db.